textile/chat-features.textile

```diff
@@ -104,121 +104,146 @@ The status of any given Chat Room is the combination of the states of all of its
-*** @(CHA-RL4b8)@ @[Testable]@ If a room lifecycle operation is not in progress, the channel state is @ATTACHED@, the room status is NOT @ATTACHED@ and all contibutors channel are now @ATTACHED@, the room status is transitioned to @ATTACHED@.
-*** @(CHA-RL4b9)@ @[Testable]@ If a room lifecycle operation is not in progress and the channel state is @SUSPENDED@, then the room status is transitioned to @SUSPENDED@, using the @reason@ of the channel state change as the error. Any transient disconnect timeouts are cancelled and the room enters the @RETRY@ loop.
-
+As well as user-initiated operations, the room must monitor its underlying resources and act accordingly. This includes handling disconnections from the server, as well as discontinuities in channel messages.
+
+* @(CHA-RL4)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL4a)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a1)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a2)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a3)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4a4)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL4b)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b1)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b2)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b3)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b4)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b5)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b6)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b7)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b8)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b9)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL4b10)@ This specification point has been removed. It was valid up until the single-channel migration.
+* @(CHA-RL11)@ The room must monitor the underlying realtime channel state, and update the room status accordingly.
+** @(CHA-RL11a)@ @[Testable]@ The room will subscribe to channel state changes on the underlying realtime channel.
```

Comment:
```suggestion
** @(CHA-RL11a)@ @[Testable]@ The room will subscribe to non-@UPDATE@ channel state events on the underlying realtime channel.
```

("channel state change" is a bit ambigious; not helped by the fact that the core SDK `ChannelStateChange` name is equally confusing because it actually represents one of two events: a state change or an UPDATE)

Reply:
02a55eff